docs/tfengine/schemas/cicd.md

```diff
@@ -121,3 +147,11 @@ Type: object
 Whether or not automatic triggering from a PR/push to branch. Default to true.
 
 Type: boolean
+
+### triggers.validate.run_on_schedule
+
+Whether or not automatic triggering according a specified schedule.
```

Comment:
This sentence seems a bit grammatically incorrect. Did you mean to say whether or not to automatically validate?

Reply:
reworded, ptal